From: Michael Hanke <mih@debian.org>
Subject: Add dependency to remote FS to LSB init header

Often condor would be installed under /usr (e.g. any system packaged
version is), and hence the daemons should not start until remote file
systems become available.
--- a/src/condor_examples/condor.boot.rpm
+++ b/src/condor_examples/condor.boot.rpm
@@ -21,8 +21,8 @@
 # LSB init part (Used by Debian package)
 ### BEGIN INIT INFO
 # Provides:          condor
-# Required-Start:    $network $local_fs
-# Required-Stop:     $local_fs $network
+# Required-Start:    $network $local_fs $remote_fs
+# Required-Stop:     $remote_fs $local_fs $network
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
 # Short-Description: Manage condor daemons
